x-pack/plugins/security/server/mocks.ts

```diff
@@ -62,4 +64,6 @@ export const securityMock = {
   createSetup: createSetupMock,
   createStart: createStartMock,
   createApiResponse: createApiResponseMock,
+  createMockAuthenticatedUser: (props: MockAuthenticatedUserProps = {}) =>
```

Comment:
I didn't have to expose this from the server mock, but I felt like this was a good way to ensure consistency between the client and server mock contracts

Reply:
Any reason why not just `createMockAuthenticatedUser: mockAuthenticatedUser`? No need to change anything, just curious.

Reply:
I debated exposing `mockAuthenticatedUser`, but prefixing it with `create` was more consistent with the other mock functions (`createSetup`, `createStart`, ...), and it felt more correct. When I first imported `mockAuthenticatedUser`, I assumed based on the name that it was an object with a `create` property or similar on it (`mockAuthenticatedUser.create()`)

Reply:
Yeah, I like the `createMockAuthenticatedUser` name, I was mostly curious if we need this additional arrow function or we can just assign `mockAuthenticatedUser` driectly to `createMockAuthenticatedUser`.

Reply:
Ah sorry I should have read your comment a little closer 🙈 

We don't need the additional arrow function, I added that so my IDE would give me type information as I wrote it out 😬